src/docs/traffic-processing-overview.md

```diff
@@ -60,7 +60,7 @@ spec:
     icap:
       - ...
         operation:
-          image: "uspregistry.azurecr.io/usp/core/waap/usp-core-waap-grpc-icap"
+          image: "uspregistry.azurecr.io/usp/core/waap/usp-core-waap-extProc-icap"
```

Comment:
shouldn't that be 

`image: "uspregistry.azurecr.io/usp/core/waap/usp-core-waap-ext-proc-icap"`